web/src/components/tasks/anytime-action/tax-clearance-certificate/AnytimeActionTaxClearanceCertificateElement.tsx

```diff
@@ -20,6 +21,7 @@ import { ReactElement, useEffect, useState } from "react";
 interface Props {
   anytimeAction: AnytimeActionLicenseReinstatement | AnytimeActionTask;
   CMS_ONLY_stepIndex?: number;
+  CMS_ONLY_downloadPage?: [];
```

Comment:
[dust]
```suggestion
  CMS_ONLY_certificatePdfArray?: [];
```

Reply:
To simplify this, I'll rename the prop and expect a boolean value `CMS_ONLY_renderDownloadPage?: boolean;` and create a new variable to manage whether is renders or not `const renderDownloadScreen = props.CMS_ONLY_renderDownloadPage || !!certificatePdfArray;`

Reply:
Great suggestion, I'll make the update, the change is 100x more clear.